File: spring-projects/springboot/1 spring-webapp/src/main/java/com/ust/webapp/model/Trainee.java
package com.ust.webapp.model;

//public record Trainee(int id, String name, String location) {
//}

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Trainee {
    private int id;
    private String name;
    private String location;
}

File: frontend/javascript typescript/Trainee.ts
interface Trainee {
    id:number;
    name:string;

    show():void;
}

let trainee:Trainee={
    id:1,
    name:"Kodi",
    show(){
        console.log(this.id)
        console.log(this.name)
    }
};

trainee.show();